Assets/Prefabs/Enemies/Chase.prefab

```diff
@@ -12,7 +12,15 @@ GameObject:
   - component: {fileID: 2499428488193469710}
   - component: {fileID: 8600112968924884190}
   - component: {fileID: 5803941238987526604}
+<<<<<<< Updated upstream
   - component: {fileID: 6891263494993504730}
+=======
+<<<<<<< HEAD
+  - component: {fileID: 1883542270689196250}
+=======
+  - component: {fileID: 6891263494993504730}
+>>>>>>> c860a7f3a26c007519422354b44caee16da94bfd
```

Comment:
WIP stash changes got into your script, your stuff is now corrupted and it's hell to take apart.